automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -266,13 +265,14 @@ trait LeonardoTestUtils
     logger.info(s"Time it took to get cluster create response with: ${clusterTimeResult.duration.toMillis} millis")
 
     // verify with get()
-    val creatingCluster = eventually {
-      verifyCluster(Leonardo.cluster.get(googleProject, clusterName),
-                    googleProject,
-                    clusterName,
-                    List(ClusterStatus.Creating),
-                    clusterRequest)
-    }(getAfterCreatePatience, implicitly[Position])
+    val creatingCluster =
+      eventually(Timeout(getAfterCreatePatience.timeout), Interval(getAfterCreatePatience.interval)) {
```

Comment:
`eventually` in new scalatest takes more implicits if we use the old format...hence using a different overloaded function that does the same thing